### PR TITLE
web/satellite: My Access Table style changes and search functionality

### PR DESCRIPTION
**Pull request by cl-mitch**
What: 

Why:

Please describe the tests:
 - Test 1:
 - Test 2:
 
Please describe the performance impact:

## Code Review Checklist (to be filled out by reviewer)
 - [x] NEW: Are there any Satellite database migrations? Are they forwards _and_ backwards compatible? 
 - [x] Does the PR describe what changes are being made?
 - [x] Does the PR describe why the changes are being made?
 - [x] Does the code follow [our style guide](https://github.com/storj/docs/blob/main/code/Style.md)?
 - [x] Does the code follow [our testing guide](https://github.com/storj/docs/blob/main/code/Testing.md)?
 - [x] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [x] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [x] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [x] Does any documentation need updating?
 - [x] Do the database access patterns make sense?

### CODE REVIEW

File: web/satellite/src/components/common/VSearch.vue [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <input
         ref="input"
         v-model="searchQuery"
-        class="common-search-input"
+        :class="`${styleType}-search-input`"
```
Comment (egonelbre) [inline web/satellite/src/components/common/VSearch.vue:8]: At this point it feels like a separate component makes more sense.

Or better yet, implement it such that it doesn't require the special case. I think the inputWidth change via code doesn't look right.

File: web/satellite/static/wasm/accessGrant.worker.js [hunk 2]
```diff
@@ -84,4 +84,4 @@ self.onmessage = async function (event) {
         default:
             self.postMessage(new Error('provided message event type is not supported'));
     }
-};
+};
```
Comment (egonelbre) [inline web/satellite/static/wasm/accessGrant.worker.js:87]: This looks like an unnecessary change.

**Review by egonelbre — COMMENTED** [review]
few nitpicks

File: web/satellite/src/utils/accessGrant.worker.js [hunk 3]
```diff
@@ -87,4 +87,4 @@ self.onmessage = async function (event) {
     default:
         self.postMessage(new Error('provided message event type is not supported'));
     }
-};
+};
```
Comment (VitaliiShpital) [inline web/satellite/src/utils/accessGrant.worker.js:90]: I'm not sure why this changes appear. Seems like one empty line at the end of the file got removed

**Review by VitaliiShpital — APPROVED** [review]

**Review by mobyvb — APPROVED** [review]
